Library/Formula/glslang.rb

```diff
@@ -0,0 +1,21 @@
+class Glslang < Formula
+  desc "glslang is an OpenGL and OpenGL ES shader front end and validator."
+  homepage "https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/"
+  url "https://github.com/KhronosGroup/glslang/archive/3.0.tar.gz"
+  sha256 "91653d09a90440a0bc35aa490d0c44973501257577451d4c445b2df5e78d118c"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+      bin.install "StandAlone/glslangValidator"
+    end
+  end
+
+  test do
+    system "#{bin}/glslangValidator", "-v"
```

Comment:
@bfontaine I've added two sample files that are bundled with glslang and run glslangValidator against these as a more robust test that the application is working: https://github.com/andystanton/homebrew/commit/150b3e9e23ee40a628131d06928a4996e54c6c0a

I've put these in ```Library/Homebrew/test/fixtures``` and attempted to follow the ```test.*``` convention. Is this an acceptable place for test fixtures?